ui-v2/app/templates/dc/acls/-form.hbs

```diff
@@ -14,7 +14,7 @@
         </div>
         <label class="type-text">
             <span>Policy <a href="{{env 'CONSUL_DOCUMENTATION_URL'}}/guides/acl.html#rule-specification" rel="help noopener noreferrer" target="_blank">(HCL Format)</a></span>
-            {{code-editor class=(if item.error.Rules 'error') name='Rules' value=item.Rules onkeyup=(action 'change')}}
+            {{code-editor class=(if item.error.Rules 'error') name='Rules' value=item.Rules syntax="hcl" onkeyup=(action 'change')}}
```

Comment:
Did you mean to include this unrelated change?

Reply:
Yes, definite decision to include it here, checked this with jack, there are going to be a few more unrelated changes to come on this also I think.